### DISCUSSION TITLE
How to toggle editable from element outside of Editable's context?

### DISCUSSION THREAD

**Discussion by benknab** [chakra-ui/chakra-ui#5707] [Q&A]
We have a card component that has multiple actions; one of them is to rename the card's title.
Is there a way to trigger the edit mode of `Editable` without wrapping the whole card (together with the actions buttons) with the `Editable` component?
We were trying to call `focus` or `click` on `Editable` or its children, but this isn't reliable sometimes; it doesn't trigger the edit mode.

**Comment by baptisteArno** [comment]
I'm facing the same issue. Did you find anything @bkiac?

**Reply by benknab** [reply]
No, we ended up wrapping the whole thing in the `Editable`

**Reply by zqin97** [reply]
Hi, I'm trying to toggle all Editable components with one button, can you share some thoughts with me?

**Reply by sophia-kravchenko** [reply]
Any progress on this? I’m trying to do the same functionality  @zqin97

**Comment by zqin97** [comment]
> Any progress on this? I’m trying to do the same functionality @zqin97

@sophia-kravchenko hello, I give up using editable and create my own custom component in the end.

**Reply by sophia-kravchenko** [reply]
Understood, thank you for your answer
